frontend/src/routes/[[lang=locale]]/candidate/(protected)/questions/+page.svelte

```diff
@@ -1,38 +1,217 @@
+
+    <Button
+      slot="primaryActions"
+      href={firstQuestionUrl}
+      variant="main"
+      icon="next"
+      text={$t('candidateApp.opinions.continue')} />
+  </BasicPage>
+{:else}
+  <BasicPage title={$t('candidateApp.questions.title')}>
+    <Warning display={!dataEditable} slot="note"
+      >{$t('candidateApp.questions.editingAllowedNote')}
+    </Warning>
+
+    <p class="pb-20 text-center">
+      {$t('candidateApp.questions.info')}
+    </p>
+    {#if nofUnansweredQuestions != 0 && !loading && dataEditable}
+      <div class="pb-6 text-center text-warning">
+        {$t('candidateApp.questions.warning', {numUnansweredQuestions: nofUnansweredQuestions})}
+      </div>
+      <div class="flex w-full justify-center pb-40 pt-20">
+        <Button
+          href={$getRoute({route: Route.CandAppQuestions, id: '1'})}
+          text={$t('candidateApp.questions.enterMissingAnswer')}
+          variant="main"
+          icon="next" />
+      </div>
+    {/if}
+
+    {#each Object.entries(questionsByCategory) as [category, categoryQuestions]}
+      <div class="edgetoedge-x">
+        <Expander
+          title={category || ''}
+          variant="category"
+          defaultExpanded={unansweredCategories?.includes(category ?? '')}>
+          {#each categoryQuestions as question}
+            <!-- Question has been answered -->
+            {#if answers?.[question.id]}
+              <div class="pb-20 pt-20">
+                <div class="text-accent">
+                  {question.category}
+                </div>
+
+                <Expander title={question.text ?? ''} variant="question">
+                  {question.info}
+                </Expander>
+
+                <div class="pt-10">
+                  <!-- This gives empty form label error from Wave Extension for every empty dot, but fix should come from LikertResponseButton -->
+                  <LikertResponseButtons
+                    name={question.id}
+                    mode="display"
+                    options={question.values}
+                    selectedKey={answers[question.id].key} />
+
+                  {#if translate(answers[question.id].openAnswer) !== ''}
+                    <div class="pt-10">
+                      <QuestionOpenAnswer
+                        >{translate(answers[question.id].openAnswer)}</QuestionOpenAnswer>
+                    </div>
+                  {/if}
+
+                  <div class="flex justify-center py-20">
+                    <Button
+                      text={getAnsweredButtonText().text}
+                      href={$getRoute({route: Route.CandAppQuestions, id: question.id})}
+                      icon={getAnsweredButtonText().icon}
+                      iconPos="left"></Button>
+                  </div>
+                </div>
+                {#if categoryQuestions[categoryQuestions.length - 1] !== question}
+                  <hr class="mt-40" />
+                {:else}
+                  <div class="mb-40" />
```

Comment:
Could the margin be assigned to some other existing element? Another option is to define more padding-bottom to the parent element. Having an empty div with margin is suspicious.